autogpt_platform/backend/backend/integrations/oauth/google.py

```diff
@@ -99,7 +149,7 @@ def _refresh_tokens(self, credentials: OAuth2Credentials) -> OAuth2Credentials:
             scopes=google_creds.scopes,
         )
 
-    def _setup_oauth_flow(self, scopes: list[str] | None) -> Flow:
+    def _setup_oauth_flow(self, scopes: List[str]) -> Flow:
```

Comment:
nit: `list` is built in since before 3.10 which is the oldest Python version we support
```suggestion
    def _setup_oauth_flow(self, scopes: list[str]) -> Flow:
```